packages/core/src/common/errors.ts

```diff
@@ -93,6 +93,10 @@ export const TOASTER_CREATE_NULL =
     ns +
     ` OverlayToaster.create() is not supported inside React lifecycle methods in React 16.` +
     ` See usage example on the docs site.`;
+export const TOASTER_CREATE_ASYNC_NULL =
+    ns +
+    ` OverlayToaster.createAsync() received a null component ref. This can happen if called inside React lifecycle ` +
+    `methods in React 16. See usage example on the docs site.`;
```

Comment:
nit: which documentation site? Blueprint or React? Might be useful to just add a link

Reply:
Agree with it being useful to just add a link. This message was copied from the `TOASTER_CREATE_NULL`. I'll update the original message as well. https://github.com/palantir/blueprint/pull/6599/commits/4779ba20a8115ac9cea8cc19c49cba263e74b1ea